### PR TITLE
Create Slidebar UI

### PR DESCRIPTION
**Pull request by DKnehi**
image

### CODE REVIEW

File: javascript-practice/src/pages/dashboard.html [hunk 1]
```diff
@@ -27,23 +31,27 @@ <h2 id="fullName" class="header-dashboard-profile-name"></h2>
...
+              src="../assets/icons/arrow.svg"
+              alt="arrow"
+            />
+            <span id="logout" class="header-dashboard-profile-sub-name logout">
+              Logout
+            </span>
           </div>
         </div>
       </header>
+      <aside class="slide-bar">
```
Comment (hoangle-agilityio) [inline javascript-practice/src/pages/dashboard.html]: slide bar??? Do you mean sidebar?

File: javascript-practice/src/pages/dashboard.html [hunk 2]
```diff
@@ -27,23 +31,27 @@ <h2 id="fullName" class="header-dashboard-profile-name"></h2>
...
-            </div>
+          </div>
+          <div class="flex-column">
+            <img
+              id="arrow"
+              class="arrow"
+              src="../assets/icons/arrow.svg"
+              alt="arrow"
+            />
+            <span id="logout" class="header-dashboard-profile-sub-name logout">
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/pages/dashboard.html]: header-dashboard-profile-sub-name??? How related between logout button and this class

File: javascript-practice/src/styles/bases/flex.css [hunk 3]
```diff
@@ -6,3 +6,6 @@
   display: flex;
   flex-direction: column;
 }
+.algin-center {
```
Comment (hoangle-agilityio) [inline javascript-practice/src/styles/bases/flex.css]: Pls careful with your typing

Comment (phuocduong-agilityio) [inline javascript-practice/src/styles/bases/flex.css]: Format

File: javascript-practice/src/pages/dashboard.html [hunk 4]
```diff
@@ -27,23 +31,27 @@ <h2 id="fullName" class="header-dashboard-profile-name"></h2>
...
+              Logout
+            </span>
           </div>
         </div>
       </header>
+      <aside class="slide-bar">
+        <img src="../assets/icons/logo.svg" alt="logo" />
+        <div class="flex algin-center dashboard-box">
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/pages/dashboard.html]: Please check design, this's not only for dashboard-box. Btw, I don't think we should use div here

File: javascript-practice/src/js/views/userView.js [hunk 5]
```diff
@@ -17,7 +17,7 @@ export default class UserView {
     this.fullNameEl = document.getElementById('fullName');
     this.headerNameEl = document.getElementById('headerName');
     this.loginFormEl = document.getElementById('login-form');
-    this.logoutEl = document.getElementById('logout');
+    this.logoutEl = document.getElementById('list-drop');
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/views/userView.js]: logoutEl and list-drop doesn't not match meaning. Btw ID should format like myCustomId, not my-custom-id

File: javascript-practice/src/js/views/userView.js [hunk 6]
```diff
@@ -66,10 +66,10 @@ export default class UserView {
   bindLogoutDropDow = () => {
     this.arrowEl.addEventListener('click', () => {
       if (this.isArrowUp) {
-        this.logoutEl.classList.add('show-logout');
+        this.logoutEl.classList.add('show-list-drop');
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/views/userView.js]: show-list-drop => Class name does not make sense

File: javascript-practice/src/pages/dashboard.html [hunk 7]
```diff
@@ -27,23 +31,35 @@ <h2 id="fullName" class="header-dashboard-profile-name"></h2>
...
+            </li>
+          </ul>
         </div>
       </header>
+      <aside class="sidebar">
+        <div class="logo-box">
+          <img src="../assets/icons/logo.svg" alt="logo" />
+        </div>
+        <ul class="sidebar-list-menu">
+          <li class="sidebar-menu menu-active">
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/pages/dashboard.html]: Li means list item, so sidebar-menu is not correct name for li element

File: javascript-practice/src/styles/utils/variables.css [hunk 8]
```diff
@@ -27,6 +27,7 @@
   --boder-radius-input-button: 8px;
   --boder-radius-login-input: 4px;
   --boder-radius-toast: 5px;
+  --boder-radius-box: 6px;
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/styles/utils/variables.css]: --boder-radius-box: 6px; => BORDER, btw boder-radius-box does not make sense, pls choose another name

Comment (phuocduong-agilityio) [reply javascript-practice/src/styles/utils/variables.css]: You can set border directly. Why use it as variable? Use variable for reuse purpose

**Pull request merged**